src/aces/aggregate.py

```diff
@@ -305,71 +315,74 @@ def aggregate_event_bound_window(
-        │ ---        ┆ ---                 ┆ ---                 ┆ ---                 ┆ ---  ┆ ---  ┆ ---  │
-        │ i64        ┆ datetime[μs]        ┆ datetime[μs]        ┆ datetime[μs]        ┆ i64  ┆ i64  ┆ i64  │
-        ╞════════════╪═════════════════════╪═════════════════════╪═════════════════════╪══════╪══════╪══════╡
-        │ 1          ┆ 1989-12-01 12:03:00 ┆ 1989-12-01 12:03:00 ┆ 1989-12-03 13:14:00 ┆ 1    ┆ 0    ┆ 0    │
-        │ 1          ┆ 1989-12-03 13:14:00 ┆ null                ┆ null                ┆ 0    ┆ 0    ┆ 0    │
-        │ 1          ┆ 1989-12-05 15:17:00 ┆ null                ┆ null                ┆ 0    ┆ 0    ┆ 0    │
-        │ 2          ┆ 1989-12-02 12:03:00 ┆ 1989-12-02 12:03:00 ┆ 1989-12-06 15:17:00 ┆ 2    ┆ 1    ┆ 0    │
-        │ 2          ┆ 1989-12-04 13:14:00 ┆ 1989-12-04 13:14:00 ┆ 1989-12-06 15:17:00 ┆ 1    ┆ 0    ┆ 0    │
-        │ 2          ┆ 1989-12-06 15:17:00 ┆ 1989-12-06 15:17:00 ┆ 1989-12-10 03:07:00 ┆ 1    ┆ 2    ┆ 1    │
-        │ 2          ┆ 1989-12-08 16:22:00 ┆ 1989-12-08 16:22:00 ┆ 1989-12-10 03:07:00 ┆ 0    ┆ 1    ┆ 0    │
-        │ 2          ┆ 1989-12-10 03:07:00 ┆ null                ┆ null                ┆ 0    ┆ 0    ┆ 0    │
-        └────────────┴─────────────────────┴─────────────────────┴─────────────────────┴──────┴──────┴──────┘
-        >>> aggregate_event_bound_window(df, ToEventWindowBounds(False, "is_C", True, None))
+        ┌────────────┬─────────────────────┬─────────────────────┬──────┬──────┬──────┬──────────────┐
+        │ subject_id ┆ timestamp_at_start  ┆ timestamp_at_end    ┆ is_A ┆ is_B ┆ is_C ┆ _EVENT_INDEX │
+        │ ---        ┆ ---                 ┆ ---                 ┆ ---  ┆ ---  ┆ ---  ┆ ---          │
+        │ i64        ┆ datetime[μs]        ┆ datetime[μs]        ┆ i64  ┆ i64  ┆ i64  ┆ i64          │
+        ╞════════════╪═════════════════════╪═════════════════════╪══════╪══════╪══════╪══════════════╡
+        │ 1          ┆ 1989-12-01 12:03:00 ┆ 1989-12-03 13:14:00 ┆ 1    ┆ 0    ┆ 0    ┆ 0            │
+        │ 1          ┆ null                ┆ null                ┆ 0    ┆ 0    ┆ 0    ┆ null         │
+        │ 1          ┆ null                ┆ null                ┆ 0    ┆ 0    ┆ 0    ┆ null         │
+        │ 2          ┆ 1989-12-02 12:03:00 ┆ 1989-12-06 15:17:00 ┆ 2    ┆ 1    ┆ 0    ┆ 1            │
+        │ 2          ┆ 1989-12-04 13:14:00 ┆ 1989-12-06 15:17:00 ┆ 1    ┆ 0    ┆ 0    ┆ 1            │
+        │ 2          ┆ 1989-12-06 15:17:00 ┆ 1989-12-10 03:07:00 ┆ 1    ┆ 2    ┆ 1    ┆ 3            │
+        │ 2          ┆ 1989-12-08 16:22:00 ┆ 1989-12-10 03:07:00 ┆ 0    ┆ 1    ┆ 0    ┆ 3            │
+        │ 2          ┆ null                ┆ null                ┆ 0    ┆ 0    ┆ 0    ┆ null         │
+        └────────────┴─────────────────────┴─────────────────────┴──────┴──────┴──────┴──────────────┘
+        >>> # unsure about event index for windows where start==end and no event (st_inclusive=False), null/#
```

Comment:
In those cases I think the window should be invalid and timestamps and event index should be null, no?